etna/insights/blocks.py

```diff
@@ -13,6 +13,9 @@
 
 class FeaturedRecordBlock(blocks.StructBlock):
     record = RecordChooserBlock()
+    image = ImageChooserBlock(
```

Comment:
Would you mind renaming to `teaser_image` to be consistent with the record embed on the results page?

https://github.com/nationalarchives/ds-wagtail/blob/5077695c52b20ed67846dbca60466df2145f9f02/etna/collections/models.py#L261-L267

Reply:
No problem.